src/components/pages/order/main/Admin/AdminPanel.js

```diff
@@ -0,0 +1,23 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import OrderContext from "../../../../../context/OrderContext";
+import { theme } from "../../../../../theme";
+import { getTabSelected, TabsConfig } from "./TabsConfig";
+
+export default function AdminPanel() {
+  const { currentTabSelected } = useContext(OrderContext) 
+  const tabs = TabsConfig
+  const TabSelected = getTabSelected(tabs, currentTabSelected)
```

Comment:
```suggestion
  const tabSelected = getTabSelected(tabs, currentTabSelected)
```
ce n'est pas un composant --> minuscule